Source/WTF/wtf/NumberOfCores.cpp

```diff
@@ -81,4 +81,16 @@ int numberOfProcessorCores()
     return s_numberOfCores;
 }
 
+#if OS(DARWIN)
+int numberOfPhysicalProcessorCores()
+{
+    int32_t val = 0;
+    size_t valSize = sizeof(val);
+    int rc = sysctlbyname("hw.physicalcpu_max", &val, &valSize, nullptr, 0);
```

Comment:
Does it make sense to use dispatch_once here to avoid the sysctlbyname call repeatedly from WebViewImpl's constructor or would we rather not have the static integer?

Or is there the concern that sysctlbyname can fail and then we would cache the wrong result?

Reply:
a static variable with a `std::call_once` sounds good to me.

Reply:
Done